railties/lib/rails/generators/rails/app/templates/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:<%= RUBY_VERSION %>
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
+RUN mkdir /<%= app_name %>
+WORKDIR /<%= app_name %>
+ADD Gemfile /<%= app_name %>/Gemfile
+ADD Gemfile.lock /<%= app_name %>/Gemfile.lock
+RUN bundle install
```

Comment:
We’ll want to install and run Yarn by default:

```dockerfile
RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
  && echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list

RUN apt-get update -qq && apt-get install -y yarn

COPY package.json yarn.lock ./
RUN yarn
```

I wonder if a first-class Rails base image is in order.